static/app/utils/replays/highlightNode.tsx

```diff
@@ -2,31 +2,31 @@ import type {Replayer} from '@sentry-internal/rrweb';
 
 const DEFAULT_HIGHLIGHT_COLOR = 'rgba(168, 196, 236, 0.75)';
 
-const highlightsByNodeId: Map<number, {canvas: HTMLCanvasElement}> = new Map();
+const highlightsByNodeIds: Map<number, {canvas: HTMLCanvasElement}> = new Map();
 const highlightsBySelector: Map<string, {canvas: HTMLCanvasElement}> = new Map();
 
 type DrawProps = {annotation: string; color: string; spotlight: boolean};
 
-interface AddHighlightByNodeIdParams extends Partial<DrawProps> {
-  nodeId: number;
+interface AddHighlightByNodeIdsParams extends Partial<DrawProps> {
+  nodeIds: number[];
 }
 interface AddHighlightBySelectorParams extends Partial<DrawProps> {
   selector: string;
 }
 
-type AddHighlightParams = AddHighlightByNodeIdParams | AddHighlightBySelectorParams;
+type AddHighlightParams = AddHighlightByNodeIdsParams | AddHighlightBySelectorParams;
 
 type RemoveHighlightParams =
   | {
-      nodeId: number;
+      nodeIds: number[];
```

Comment:
Will the new sdk version break the UI in self hosted sentry sending data on this old version/format?

Reply:
Looks like this method isn't called directly with data from the sdk, instead something like static/app/utils/replays/hooks/useCrumbHandlers.tsx is fixing the data first with code like this, so we should be good.:
```
  if ('nodeId' in data) {
    return {nodeIds: [data.nodeId], annotation: record.data.label};
  }
  if ('nodeIds' in data) {
    return {nodeIds: data.nodeIds, annotation: record.data.label};
  }
```

Reply:
a better example is: `getNodeIds` in static/app/utils/replays/types.tsx